### DISCUSSION TITLE
Magick convert PNG to DDS using DXT3

### DISCUSSION THREAD

**Discussion by Paradoxu** [ImageMagick/ImageMagick#6690] [Contribute]
### ImageMagick version

7.1.0-26

### Operating system

Windows

### Operating system, version and so on

Windows 10

### Description

I'm trying to use the following command, to convert a PNG image previously created with `ImageMagick` back to it's original `.dds` file, the problem is that the output file is always using DXT5 even tho I'm passing the compression value as DXT3: 

> magick convert ./myImage.png -define dds:compression=DXT3 'DDS:myImage.dds'

But when I open the output  `myImage.dds` a se on the headers that it is using DXT5, am I passing the wrong commands? Sadly I can't use formats other than DXT3. I saw on some old questions in forums, people saying that `ImageMagick` doesn't support DXT3 yet, but like 3 years ago if not more... When I use the command the get the list of convert formats I see DXT3.

### Steps to Reproduce

- From a DXT3 sample create a PNG file
- From the output PNG file try to reconvert back to .dds

### Images

![image](https://user-images.githubusercontent.com/25586710/155499288-0fc4d3bf-9309-4fa8-8423-27b44ebe4fd3.png)

**Comment by dlemstra** [comment]
There is no support for writing DXT3 DDS files.
